spanner/client.go

```diff
@@ -713,6 +732,19 @@ func (w *wrappedStream) RecvMsg(m any) error {
 	if mt.currOp.currAttempt != nil {
 		mt.currOp.currAttempt.setDirectPathUsed(isDirectPathUsed)
 	}
+	if header, err := w.ClientStream.Header(); err == nil {
```

Comment:
Do we receive this for every message? Or only at the start of the stream? In the latter case, this should probably move to the initialization function for the stream.

Reply:
This will only be sent in the initial metadata so only need to process once.

Reply:
Thanks, moved this out from RecvMessage, added the logic to set server-timing for streaming request in grcp_client.go and upon recordAttemptCompletion only we will record GFE/AFE metrics